### PR TITLE
frontend/feat: Added Bike Assets and unified lotties

### PR DESCRIPTION
**Pull request by slow-codex**
## Type of Change

- [x] Bugfix
- [ ] New feature
- [x] Enhancement
- [x] Refactoring
- [ ] Dependency updates

## Description

- Moved Bike assets from Juspay to MovingTech asset store.
- Unified all the ride_accepted, waiting_for_customer and accepted_by_another_driver lotties removing merchant and variant checks.

### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added integration tests for my changes where possible
- [ ] I tested the changes using integration tests
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable
- [ ] No leak detected in leakcanary

### CODE REVIEW

**Review by slow-codex — COMMENTED** [review]
Main PR - https://github.com/nammayatri/nammayatri/pull/9011

File: Frontend/android-native/app/src/driver/jatriSaathiDriver/res/raw/ride_accepted_lottie.json
Comment (spravinkumar9952) [inline Frontend/android-native/app/src/driver/jatriSaathiDriver/res/raw/ride_accepted_lottie.json:1]: Compress all lottie files using lottie compresser

**Review by spravinkumar9952 — CHANGES_REQUESTED** [review]

File: Frontend/android-native/app/src/driver/jatriSaathiDriver/res/raw/ride_accepted_lottie.json
Comment (spravinkumar9952) [reply Frontend/android-native/app/src/driver/jatriSaathiDriver/res/raw/ride_accepted_lottie.json:1]: In the hot push Lottie files won't go. is this files are updated in asset store?

Comment (slow-codex) [reply Frontend/android-native/app/src/driver/jatriSaathiDriver/res/raw/ride_accepted_lottie.json:1]: And the `nav_on_map_icon` is already added in asset store but that also is used in native side so native update is required.

**Review by ashishsingh101 — APPROVED** [review]

**Review by spravinkumar9952 — APPROVED** [review]

**Pull request merged**